addons/io_scene_gltf2/__init__.py

```diff
@@ -107,6 +107,20 @@ def __init__(self):
         default=''
     )
 
+    export_image_format = EnumProperty(
+        name='Image Format',
+        items=(('JPEG', 'jpeg image format (.jpg)',
+                'encode and save textures as .jpeg files. Be aware of a possible loss in quality.'),
+               ('PNG', 'png image format (.png)',
+                'encode and save textures as .png files.')
+               ),
+        description=(
+            'Output format for images. PNG is lossless and generally preferred, but JPEG might be preferable for web '
+            'applications due to the smaller file size'
+        ),
+        default='PNG'
+    )
```

Comment:
We circled around different ways of doing this in glTF-Blender-Exporter. In https://github.com/KhronosGroup/glTF-Blender-Exporter/pull/181 I tried to be clever about using presence or absence of alpha to determine output format, which we eventually reverted in favor of https://github.com/KhronosGroup/glTF-Blender-Exporter/pull/221, to simply have export match the input format wherever possible.

In hindsight, I think matching the input format is the best approach. Consider:

- Exporting a JPEG source image as JPEG should not result in any loss of quality.
- Users retain greater control of their textures, because they can choose format and compression settings before bringing them into Blender.
- Users may not want the same format for all images. In particular, base color should almost always use JPEG on web, whereas normal maps should almost always use PNG.

For the reasons above, I'm not sure we should expose a UI checkbox for the format of all textures, and if we do, "Source format" should be the default option.